app/Services/AuthenticationService.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brewmap\Services;
+
+use Brewmap\Eloquent\User;
+use Illuminate\Http\Exceptions\HttpResponseException;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
+use Laravel\Socialite\Contracts\User as FacebookUser;
+
+class AuthenticationService
+{
+    /**
+     * @throws HttpResponseException
+     */
+    public function login(Request $request): string
+    {
+        $user = User::where("email", $request->email)->first();
+
+        if ($user === null || !Hash::check($request->password, $user->password)) {
```

Comment:
Please do not use facades, especially in service layer. There's a [facade class reference section](https://laravel.com/docs/8.x/facades#facade-class-reference) in Laravel docs. I'll add this to code conventions in #26.